src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataAggregatorLambda.java

```diff
@@ -146,13 +150,15 @@ public String PerformFunction(String input, Context context) throws Exception {
         boolean isIsoFile = (iso != null);
 
         mtfe = new MetadataFilesToEcho(isIsoFile);
-        mtfe.setDatasetValues(collectionName, collectionVersion, rangeIs360, boundingBox);
+        mtfe.setDatasetValues(collectionName, collectionVersion, rangeIs360, boundingBox, additionalAttributes);
         if (granules != null && granules.size() > 0) {
             mtfe.setGranuleFileSizeAndChecksum(granules);
         }
 
 		if (isIsoFile) {
 			try {
+				// Reason for giving a list of files: to check which ones are PNG's so we can add in extra metadata
```

Comment:
Should we remove this command and the one below (line 171-172) since we are no longer checking for pngs?

Reply:
Removed